[Blazor] Blazor WebAssembly perf benchmarks are failing

The Docker image is failing to build with the following message:

```text
/src/src/Components/Endpoints/src/Microsoft.AspNetCore.Components.Endpoints.csproj(67,5): warning : '/src/src/Components/Endpoints/src/../../Web.JS/dist/Release/blazor.web.js' does not exist. Ensure the JS assets have been build by running 'npm run build' from the repository root.
CSC : error CS1566: Error reading resource '_framework/blazor.web.js' -- 'Could not find a part of the path '/src/src/Components/Web.JS/dist/Release/blazor.web.js'.' [/src/src/Components/Endpoints/src/Microsoft.AspNetCore.Components.Endpoints.csproj]
```

The fix seems straightforward enough - add an instruction to run `npm run build` after:https://github.com/dotnet/aspnetcore/blob/d64b69df5c85464809f19bf878a9b54bc0519a23/src/Components/benchmarkapps/Wasm.Performance/dockerfile#L26

--- a/src/Components/benchmarkapps/Wasm.Performance/dockerfile
+++ b/src/Components/benchmarkapps/Wasm.Performance/dockerfile
@@ -24,6 +24,7 @@ RUN git init \
     && git remote add origin https://github.com/aspnet/aspnetcore
 
 RUN ./restore.sh
+RUN npm run build
 RUN .dotnet/dotnet publish -c Release --no-restore -o /app ./src/Components/benchmarkapps/Wasm.Performance/Driver/Wasm.Performance.Driver.csproj
 RUN chmod +x /app/Wasm.Performance.Driver
 

